modules/apps/27-interchain-accounts/module_test.go

```diff
@@ -0,0 +1,75 @@
+package ica_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+	//	abci "github.com/tendermint/tendermint/abci/types"
```

Comment:
Is it safe to remove this?

Reply:
yes! leftover code from testing. Nice catch